config/webpack.constants.js

```diff
@@ -14,41 +14,6 @@ const FOAM_BIN_TMP_DIR = `${ROOT_DIR}/.local`;
 const FOAM_BIN_TMP_PATH = `${FOAM_BIN_TMP_DIR}/foam-bin.js`;
 const FOAM_BIN_REG_EXP = /foam-bin\.js$/;
 const ES6_REG_EXP = /\.es6\.js$/;
-const ES6_LOADER_OPTIONS_PROD = {
-  plugins: [
-    // presets: ['es2015'] without strict mode:
-    'check-es2015-constants',
-    'transform-es2015-arrow-functions',
-    'transform-es2015-block-scoped-functions',
-    'transform-es2015-block-scoping',
-    'transform-es2015-classes',
-    'transform-es2015-computed-properties',
-    'transform-es2015-destructuring',
-    'transform-es2015-duplicate-keys',
-    'transform-es2015-for-of',
-    'transform-es2015-function-name',
-    'transform-es2015-literals',
-    'transform-es2015-object-super',
-    'transform-es2015-parameters',
-    'transform-es2015-shorthand-properties',
-    'transform-es2015-spread',
-    'transform-es2015-sticky-regex',
-    'transform-es2015-template-literals',
-    'transform-es2015-typeof-symbol',
-    'transform-es2015-unicode-regex',
-    'transform-regenerator',
-    'transform-es2015-block-scoping',
-    'transform-es2015-arrow-functions',
-    'transform-es2015-template-literals',
-    ['transform-es2015-modules-commonjs', {strict: false}],
-  ],
-  // No, really. Without strict mode.
-  parserOpts: {strictMode: false},
-};
-const ES6_LOADER_OPTIONS_DEV = {
-  plugins: ES6_LOADER_OPTIONS_PROD.plugins.concat(['transform-runtime']),
```

Comment:
The code coverage has changed and it comes down mostly this line. On master, just commenting out this line but still using babel causes the number of lines to decrease. Not using babel at all (as in this PR) decreases the number functions/etc. too.

I suspect that because of the transformation some of the coverage information will have been bogus, but will have to check a bit more.

Reply:
Addressed by landing https://github.com/GoogleChromeLabs/confluence/pull/398 first.